__tests__/components/graph-view-v2.test.js

```diff
@@ -0,0 +1,1280 @@
+// @flow
+
+import * as d3 from 'd3';
+import * as React from 'react';
+import { createRoot } from 'react-dom/client';
+import { shallow } from 'enzyme';
+import Background from '../../src/components/background';
+import Defs from '../../src/components/defs';
+import GraphUtils from '../../src/utilities/graph-util';
+import GraphViewV2 from '../../src/components/graph-view-v2';
+
+jest.mock('react-dom/client', () => ({
+  ...jest.requireActual('react-dom/client'),
+  createRoot: jest.fn(),
+}));
+
+describe('GraphViewV2 component', () => {
```

Comment:
The change affect 11 tests. 1 of them is that we don't trigger `createRoot`. 10 of them are about the parameter. Since we are adding forceUpdate,  the example below has to take two more parameters.
```
expect(instance.renderEdge).toHaveBeenCalledWith(
        'edge-a-b',
        'blah',
        expectedEdge,
        false,
        false
      );
```